agents1/OfficialAgent.py

```diff
@@ -658,10 +674,11 @@ def decide_on_actions(self, state):
                     self.received_messages = []
                     self.received_messages_content = []
                     # Decrement willingness because agent is lying about victim locations
-                    trustBeliefs[self._humanName].decrement_trust(0.05)
+                    trustBeliefs[self._humanName].decrement_willingness(0.3)
```

Comment:
Should probably decrease competence a bit too right (not 0.3 though)? Could also be because of human memory